mw-webapp/src/index.scss

```diff
@@ -10,6 +10,18 @@ body {
   background-color: var(--primaryBackgroundColor);
   background-color: var(--primaryBackgroundColor);
   color: var(--primaryTextColor);
+  font-family: "Roboto", sans-serif;
+  font-size: $fontSizeBig;
+}
+
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+button {
+  font-family: "Roboto", sans-serif;
+  font-size: $fontSizeBig;
```

Comment:
are you sure? I recommend to avoid global styles

Reply:
yes, because button vy default has another font-family and size 
![image](https://github.com/tritonJS826/masters-way/assets/112805668/757a2554-1960-4c98-839e-eb05d37981db)

Reply:
let's define mixin for such purpose in the variables and reuse it if possible

Reply:
add font-family to variable $primaryFontFamily